index.html

```diff
@@ -14,6 +14,66 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+  <header>  
+    <h1>Natalie Peterson</h1>
+    <h2>child & friend</h2>
+  <nav>
+    <a href="#">About</a>
+    <a href="https://objective-bose-29c02b.netlify.app/">Portfolio</a>
+    <a href="#">Contact</a>
+  </nav>
+</header>
+<section>
+  <div>
+  <h3>Experience</h3>
+  <h5>Cookie Connoisseur</h5>
+  <h6>feb 1996 - current</h6>
```

Comment:
You'll have to check, but I think using `h5` and `h6` levels are generally frowned as being too granular. I don't know for sure, but I remember reading that somewhere and hardly use them.